quarkus/addons/source-files/deployment/src/main/java/org/kie/kogito/addon/source/files/deployment/KogitoAddOnSourceFilesProcessor.java

```diff
@@ -63,4 +75,31 @@ void addSourceFileProcessBindListener(KogitoBuildContextBuildItem ctxBuildItem,
         kogitoBuildContext.getSourceFileCodegenBindNotifier()
                 .ifPresent(notifier -> notifier.addListeners(processListener, serverlessWorkflowListener));
     }
+
+    @BuildStep
+    NativeImageResourceBuildItem nativeImageResourceBuildItem(KogitoBuildContextBuildItem ctxBuildItem) throws IOException {
+        List<String> sourceFiles = getSourceFiles(ctxBuildItem.getKogitoBuildContext().getAppPaths().getResourceFiles());
+        return new NativeImageResourceBuildItem(sourceFiles);
```

Comment:
My one line nitpick ;) thanks a lot Helber!!!
```suggestion
        return new NativeImageResourceBuildItem(getSourceFiles(ctxBuildItem.getKogitoBuildContext().getAppPaths().getResourceFiles()));
```